lib/core/jdl_application_configuration.js

```diff
@@ -100,8 +109,19 @@ function filterOutOptionsWithoutValues(options) {
 }
 
 function filterOutOptionsThatShouldNotBeExported(options) {
-  const filteredOptions = { ...options };
+  const filteredOptions = options;
```

Comment:
The reason behind the previous code was to actually duplicate the passed options and not modify it directly to avoid side-effects.

Reply:
Yes, it's a local function and already is done at the parent function filterOutUnwantedOptions.